src/server/host/Endpoints/Administration/Users/CreateUserEndpoint.cs

```diff
@@ -51,18 +55,30 @@ public CreateUserRequestValidator(IIdService idService)
 public class CreateUserEndpoint(
     DatabaseContext databaseContext,
     IRealtimeEventsService realtimeEventsService,
-    IIdService idService
+    IIdService idService,
+    IUserMapper userMapper
 ) : Endpoint<CreateUserRequest, CreateUserResponse>
 {
     public override void Configure()
     {
         Post("");
         Group<UserAdministrationGroup>();
         Description(x => x.ClearDefaultProduces(200).Produces<CreateUserResponse>(201));
+        this.ProducesError(EndpointErrors.UserExists);
     }
 
     public override async Task HandleAsync(CreateUserRequest req, CancellationToken ct)
     {
+        var existentUser = await databaseContext
```

Comment:
Maybe you can additionally add a unique index on the `Alias` column, so it is also checked in the database

Reply:
good point - but I want to be more flexible if at some point I really need two users with the same name - Then it would be locked in the database - or is it possible to revert the lock entry?

Reply:
sure, you can remove the index later again. But I guess you can also just add some additional context to an alias if two have the same name.